docs/guide/data_migration/importing_data.md

```diff
@@ -0,0 +1,165 @@
+# Importing data
+
+To import Repository data from YAML files, run the `ibexa:migrations:migrate` command.
```

Comment:
Strictly speaking this is correct, although a different command exists (`ibexa:migrations:import` that allows "importing" files from elsewhere in the system into Migrations directory). Describing both as import might be confusing for the user.

Therefore, to more clearly show what data is manipulated, I'd suggest:
```suggestion
To import data from YAML migration files into Repository, run the `ibexa:migrations:migrate` command.
```

This comment might also affect the paragraph below, as ideally one would use the aforementioned command to import migration files, as it handles picking the right directory for him/her.